README.md

```diff
@@ -3,10 +3,10 @@ Online implementation of Hi-Lo.
 
 ## Game Description
 
-N players play the game; best played if $N > 3, N < 8.$ N cards are dealt randomly from a standard deck, one to each player.
+N players play the game; the game is best played if there are between 3 and 8 players. N cards are dealt randomly from a standard deck, one to each player.
 
-If there are any duplicates at the start of the game, the number of duplicate cards are identified -- for example, 1 group of 3 duplicates, or 2 groups of 2 duplicates, and given as information to all the players.
+If there are any duplicates at the start of the game, the number of duplicate cards are identified -- for example, 1 group of 3 duplicates, or 2 groups of 2 duplicates. This information is revealed to all the players.
 
-There are two rounds to the game. The first round is as follows: going around the circle, the players each state which numerical rank $1,\dots,N$ they think they are. Each of the other players can see others' cards, **but not their own.** They also have access to the other players' guesses about their ranks.
+There are two rounds to the game. The first round is as follows: going around the circle, the players each state which numerical rank (from 1 to N) they think they are. Each of the other players can see the others' cards, **but not their own.** They also have access to the other players' guesses about their ranks.
```

Comment:
is 1 the highest or lowest?

Reply:
Based on prev experience I think 1 is the highest ranked card, N would be the lowest. For example, a player with an Ace would always be first; a player with a two would always be last or tied for such